litert/vendors/qualcomm/core/common.h

```diff
@@ -9,6 +9,12 @@
 extern "C" {
 #endif  // __cplusplus
 
+typedef enum LiteRtProfilingOptions {  // NOLINT(modernize-use-using)
+  kProfilingOff = 0,
+  kProfilnigBasic = 1,
+  kProfilingDetailed = 2
```

Comment:
If you don't "namespace" these values, you are basically preventing user code to reuse these simple names in their code.

The 2nd value has a typo: `Profilnig` -> `Profiling` 

```suggestion
  kLiteRtProfilingOff = 0,
  kLiteRtProfilingBasic = 1,
  kLiteRtProfilingDetailed = 2
```

The same comment applies to the `LiteRtQnnLogLevel` enum below.